src/fullfile.c

```diff
@@ -99,6 +99,15 @@ static int download_loop(void *download_handle, struct list *files, int *num_dow
 	return swupd_curl_parallel_download_end(download_handle, num_downloads);
 }
 
+static bool download_error(int response, UNUSED_PARAM void *data)
+{
+	if (response == 404) {
```

Comment:
why not just `return (response == 404);` ?

Reply:
good point.